versions/1/changelog.yml

```diff
@@ -2,11 +2,11 @@
 ## This file documents changes in the package specification. It is NOT a package specification file.
 ## Newer entries go at the bottom.
 ##
-- version: 1.1.1-next
+- version: 1.2.0
   changes:
   - description: Prepare v1.1.1-next
     type: enhancement
-    link: https://github.com/elastic/package-spec/pull/?
+    link: https://github.com/elastic/package-spec/pull/219
```

Comment:
Umm, does this change need to be mentioned?

Reply:
Done